<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CashierDialog</class>
 <widget class="QDialog" name="CashierDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1261</width>
    <height>772</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>80</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>首页</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>80</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>注销</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_4">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>80</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>锁屏</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="exitSystemButton">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>80</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>退出系统</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>营业员:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="cashierNameLineEdit">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>销售日期:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="sellDate">
         <property name="date">
          <date>
           <year>2018</year>
           <month>7</month>
           <day>13</day>
          </date>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>销售单号:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="sellOrderNumber">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>会员号:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="memberNumber">
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="searchMember">
         <property name="text">
          <string>查找</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableView" name="dataTableView">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>300</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_4">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>150</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="title">
          <string>商品销售方案</string>
         </property>
         <widget class="QCheckBox" name="checkBox">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>20</y>
            <width>91</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>启用 折扣</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="checkBox_2">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>40</y>
            <width>71</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>启用 满减</string>
          </property>
         </widget>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBox_5">
         <property name="maximumSize">
          <size>
           <width>500</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="title">
          <string>在下框中输入&quot;商品条形码&quot;</string>
         </property>
         <widget class="QLineEdit" name="barcodeLineEdit">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>50</y>
            <width>181</width>
            <height>31</height>
           </rect>
          </property>
         </widget>
         <widget class="QPushButton" name="searchGoodPushButton">
          <property name="geometry">
           <rect>
            <x>220</x>
            <y>50</y>
            <width>71</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>查找</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>110</y>
            <width>151</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>条形码扫描器: 扫描中...</string>
          </property>
         </widget>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QGroupBox" name="groupBox_6">
         <property name="title">
          <string>结算:</string>
         </property>
         <widget class="QPushButton" name="settlementPushButton">
          <property name="geometry">
           <rect>
            <x>410</x>
            <y>30</y>
            <width>91</width>
            <height>91</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>结算</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="totalPriceEditLine">
          <property name="geometry">
           <rect>
            <x>72</x>
            <y>39</y>
            <width>111</width>
            <height>41</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLabel" name="label_6">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>41</y>
            <width>54</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>总价:</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_8">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>50</y>
            <width>20</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>￥</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="moneyInLineEdit">
          <property name="geometry">
           <rect>
            <x>260</x>
            <y>20</y>
            <width>113</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>14</pointsize>
           </font>
          </property>
         </widget>
         <widget class="QLabel" name="label_7">
          <property name="geometry">
           <rect>
            <x>210</x>
            <y>20</y>
            <width>51</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>实收:</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_9">
          <property name="geometry">
           <rect>
            <x>210</x>
            <y>80</y>
            <width>51</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>找零:</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="moneyOutLineEdit">
          <property name="geometry">
           <rect>
            <x>260</x>
            <y>80</y>
            <width>113</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
